server/pxf-hdfs/src/main/java/org/greenplum/pxf/plugins/hdfs/orc/ORCVectorizedAccessor.java

```diff
@@ -57,6 +57,7 @@
     private static final TreeTraverser TRAVERSER = new TreeTraverser();
 
     static final String MAP_BY_POSITION_OPTION = "MAP_BY_POSITION";
+    private static final String UNSUPPORTED_ERR_MESSAGE = "Write operation is not supported";
```

Comment:
```suggestion
    private static final String UNSUPPORTED_ERR_MESSAGE = "Write operation is not supported for vectorized ORC profile";
```
or maybe something along these lines?